sdks/python/apache_beam/examples/cookbook/datastore_wordcount.py

```diff
@@ -246,10 +247,10 @@ def run(argv=None):
   result = read_from_datastore(gcloud_options.project, known_args,
                                pipeline_options)
 
-  empty_line_values = result.aggregated_values(empty_line_aggregator)
-  logging.info('number of empty lines: %d', sum(empty_line_values.values()))
-  word_length_values = result.aggregated_values(average_word_size_aggregator)
-  logging.info('average word lengths: %s', word_length_values.values())
+  result.metrics().query()
+  #TODO(pabloem): Fix these once metrics are 100% queriable.
```

Comment:
Add a JIRA issue, and I do no think you need to keep commented out lines here. Just say that after fixing the bug print metrics information maybe?